package-lock.json

```diff
@@ -34,7 +34,6 @@
 				"@nestjs/swagger": "^6.2.1",
 				"@types/cache-manager-redis-store": "^2.0.1",
 				"@types/connect-redis": "^0.0.19",
-				"@types/connect-redis": "^0.0.19",
```

Comment:
what happened here?

Reply:
I haven't edited package-lock manually. As i can see those dependencies were added by Maximilian Bischof and dependabot at the same day. I think while doing npm install locally npm deleted unnecessary repetition.
![image](https://user-images.githubusercontent.com/118356546/226284698-389ab0d6-39c2-457f-addf-1691bc042274.png)

Reply:
run npm ci instead of npm install